examples/src/main/java/io/grpc/examples/experimental/CompressingHelloWorldServerAllMethods.java

```diff
@@ -62,15 +63,19 @@ private void start() throws IOException {
       public void run() {
         // Use stderr here since the logger may have been reset by its JVM shutdown hook.
         System.err.println("*** shutting down gRPC server since JVM is shutting down");
-        CompressingHelloWorldServerAllMethods.this.stop();
+        try {
+          CompressingHelloWorldServerAllMethods.this.stop();
+        } catch (InterruptedException e) {
+          e.printStackTrace();
```

Comment:
Couple of comments:
- do you want to print to stderr in keeping with lines 65 and 71?
`e.printStackTrace(System.err)`

- do you want to mention/comment saying this is because the logger is closed as part of shutdown?

and of course make this change everywhere.

Reply:
> do you want to print to stderr in keeping with lines 65 and 71?
e.printStackTrace(System.err)

Done in https://github.com/grpc/grpc-java/pull/6512/commits/d997e5be9504f61d4f9fe23998cb400899a95274

I also went ahead and fixed the call to `logger` in `ManualFlowControlServer`

> do you want to mention/comment saying this is because the logger is closed as part of shutdown?

The call to `System.err.println` above already has this comment 😄